<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenerateRoutesUi</class>
 <widget class="QDialog" name="GenerateRoutesUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>706</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rute</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButtonCost">
       <property name="text">
        <string>COST</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonInit">
       <property name="text">
        <string>INIT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonOne">
       <property name="text">
        <string>ONE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonAll">
       <property name="text">
        <string>ALL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonSave">
       <property name="text">
        <string>SAVE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonReset">
       <property name="text">
        <string>RESET</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Working Day(Hours)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineHours">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelInitializing">
       <property name="text">
        <string>Initializing..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarInitializing">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>20</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolButtonOne</sender>
   <signal>clicked()</signal>
   <receiver>GenerateRoutesUi</receiver>
   <slot>OnButtonOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>172</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonAll</sender>
   <signal>clicked()</signal>
   <receiver>GenerateRoutesUi</receiver>
   <slot>OnButtonAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonInit</sender>
   <signal>clicked()</signal>
   <receiver>GenerateRoutesUi</receiver>
   <slot>OnButtonInit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>46</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonCost</sender>
   <signal>clicked()</signal>
   <receiver>GenerateRoutesUi</receiver>
   <slot>OnButtonCost()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>GenerateRoutesUi</receiver>
   <slot>OnButtonSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>208</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButtonReset</sender>
   <signal>clicked()</signal>
   <receiver>GenerateRoutesUi</receiver>
   <slot>OnButtonReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>259</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>382</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnButtonOne()</slot>
  <slot>OnButtonAll()</slot>
  <slot>OnButtonInit()</slot>
  <slot>OnButtonCost()</slot>
  <slot>OnButtonSave()</slot>
  <slot>OnButtonReset()</slot>
 </slots>
</ui>
